### ISSUE TITLE
Hardware: Reverse voltage protection

### ISSUE THREAD

**Issue by psykauze** [DragonShadesX/Arduino-Laser-Tag#7]
I've seen two diodes at the Power connector. If the purpose of those diode is to protect from a reverse polarity, one diode is enough.

You could also use a "poka-yoke" connector to prevent users errors.

Also, "I think" all the parts used on the board is y et reversed protected (Teensy, Leds, etc...).

**Comment by DragonShadesX** [comment]
Yes we could probably use one diode, yes we could use a connector but the idea was to solder on a 9V battery connector so we didn't bother to add connector.  
Not sure about the reverse protection, but either way too high voltage applied backwards will fry/brick them so we decided to make it foolproof.

**Comment by psykauze** [comment]
What do you mean by "too high voltage applied backwards" ?

Do you mean you want to protect the battery of reverse current ? In this case, I still think one diode is enough and you'll gain some precious Watts (and battery capacity).

Note: Oh great, the 9V input is only connected to a voltage regulator. If this voltage regulator is directly soldered to the board, it does also the reverse protection.

**Comment by DragonShadesX** [comment]
One is probably enough, my thinking was that I wanted to make it foolproof so if someone connected the battery backwards the instant potential across the LEDs/powering the Teensy in the wrong direction couldn't damage anything.  I already bricked two Arduinos in other projects under normal usage (probably a bad power supply) and I didn't have time to get spares, so I didn't want to take chances.  
The other reason is because this whole project was put together by a few people who had never really done this before, so I wanted to build in some safeties for if/when something went wrong (especially near the batteries in case we switched to LiPos).  For a final version?  Might take it out after I do some more research.  For a prototype and learning experience?  I had a bunch of those diodes on hand and it didn't cost me anything, so I figured why not.

**Comment by psykauze** [comment]
Ok, not a problem. It's not a real issue because it should work like this. I'm just used to see only one diode to prevent "reverse polarity" issues. It's only because you will have a 2V Vdrop.

Also 1N4001 diodes are not very good (but very cheap). If you want to tune this part, you can use a Schottky rectifier diode as a replacement.
